tests/integration/controller/kyma/manifest_test.go

```diff
@@ -7,6 +7,8 @@ import (
 	"reflect"
 	"strings"
 
+	"github.com/kyma-project/lifecycle-manager/pkg/templatelookup"
```

Comment:
Please rearrange the imports according to the following:
  gci:
    sections:
      - standard                                             # Standard packages.
      - default                                              # Imports that could not be matched to another section type.
      - prefix(github.com/kyma-project/lifecycle-manager)    # Imports with the specified prefix.
      - blank                                                # Blank imports.
      - dot